pages/treeherder.py

```diff
@@ -29,6 +29,7 @@ class TreeherderPage(Base):
     _next_twenty_locator = (By.CSS_SELECTOR, 'div.btn:nth-child(2)')
     _next_fifty_locator = (By.CSS_SELECTOR, 'div.btn:nth-child(3)')
     _notification_locator = (By.CSS_SELECTOR, 'ul#notification-box')
+    _resultset_visible_locator = (By.CSS_SELECTOR, 'span.hidden.ready')
```

Comment:
You are correct- I didn't see it because of the line the comment was on. I will fix it [I like alphabetical also!]